docs/env.md

```diff
@@ -1,117 +1,258 @@
 
-Default
+- Type: string
+- Required: Yes
+- Default: None
 
-:   `sqlite:///path/to/project/db.sqlite3` if `DJANGO_CONFIGURATION` is
-    set to `Development`, None otherwise
+#### DJANGO_AWS_DEFAULT_REGION
 
-Choices
+The Amazon Web Services region where the relevant AWS deployment is located.
 
-:   See [schemas as presented by
-    dj-database-url](https://github.com/kennethreitz/dj-database-url#url-schema)
+- Type: string
+- Required: No
+- Default: `"eu-west-1"`
+- Choices: Any valid AWS region name.
+
+#### DJANGO_AWS_SOURCE_BUCKET_NAME
+
+The source AWS S3 bucket where files will be uploaded by end users. This should match the name of the bucket created by the relevant AWS deployment.
+
+- Type: string
+- Required: No
+- Default: Varies depending on the environment.
+  - `"development-marsha-source"` in development;
+  - `"test-marsha-source"` in test;
+  - `"staging-marsha-source"` in staging;
+  - `"preprod-marsha-source"` in preproduction;
+  - `"production-marsha-source"` in production.
+
+#### DJANGO_CLOUDFRONT_ACCESS_KEY_ID
+
+ID for the access key used to sign Cloudfront URLs. Must be associated with a master account as IAM accounts cannot sign URLs.
+
+- Type: string
+- Required:
+  - Yes when `DJANGO_CLOUDFRONT_SIGNED_URLS_ACTIVE` is `True`;
+  - No otherwise.
+- Default: None;
+
+#### CLOUDFRONT_PRIVATE_KEY_PATH
+
+Path to a private key corresponding to the id in `DJANGO_CLOUDFRONT_ACCESS_KEY_ID`. Also used to sign Cloudfront URLs.
+
+- Type: string
+- Required:
+  - Yes when `DJANGO_CLOUDFRONT_SIGNED_URLS_ACTIVE` is `True` and the key is not located in the default path;
+  - No otherwise.
+- Default: `{REPO_ROOT}/.ssh/cloudfront_private_key`
+
+#### DJANGO_CLOUDFRONT_SIGNED_URLS_ACTIVE
+
+Whether Cloudfront URLs for MP4 files and subtitle tracks should be cryptographically signed.
+
+Note: Preview images are never signed as a matter of policy; adaptive streaming formats pose technical challenges when it comes to signed URLs, so we're not doing any signing there for now.
+
+- Type: Boolean
+- Required: No
+- Default: Varies depending on the environment:
+  - `False` in development and test;
+  - `True` in all other environments.``
+- Choices: `True` or `False`
+
+#### DJANGO_CLOUDFRONT_URL
+
+The URL for the AWS Cloudfront distribution for the relevant AWS deployment. This is the domain that will be used to distribute processed files to end users.
+
+- Type: string
+- Required: Yes
+- Default: None
+
+
+## 2. AWS deployment environment
+
+Specified in `{REPO_ROOT}/src/aws/env.d/development`.
+
+#### AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY
+
+A key ID + secret pair for an AWS IAM account with administrative access to the resources the AWS deployment we're operating on is using.
+
+- Type: string
+- Required: Yes
+- Default: None
+
+#### TF_VAR_aws_region
+
+The Amazon Web Services region where the AWS deployment we're operating on is located.
```

Comment:
I would say: to which we want to deploy our serverless stack or something like this?